app/javascript/controllers/actors_controller.js

```diff
@@ -0,0 +1,40 @@
+import { Controller } from 'stimulus'
+import axios from 'axios'
+import { csrfToken, jumpToCreator } from './stimulus_modules'
+
+export default class extends Controller {
+  // Intercept Rails UJS's ajax:success callback and update the modal with the resulting content.
+  // This avoids having to have a separate .js.erb partial in a typical Rails AJAX setup.
+  success (event) {
+    const [data, status, xhr] = event.detail // eslint-disable-line no-unused-vars
+    document.getElementsByClassName('modal-content').item(0).innerHTML = data.body.innerHTML
+  }
+
+  append (event) {
+    event.preventDefault()
+    const data = { actor_id: event.target.dataset.id }
+    data.index = document.getElementsByClassName('nested-fields').length
+
+    axios({
+      method: 'POST',
+      url: event.target.dataset.url,
+      data: data,
+      headers: {
+        'X-CSRF-Token': csrfToken()
+      }
+    })
+      .then(function (response) {
+        document
+          .getElementById('creator_aliases')
```

Comment:
Might be a cool idea to pass the `'creator_aliases'` id in as a data-attribute and really decouple this logic from the html